evaluate/metrics/assessment_test.go

```diff
@@ -193,20 +196,103 @@ func TestFormatStringCSV(t *testing.T) {
 				AssessmentKeyCoverageStatement: 1,
 				AssessmentKeyFilesExecuted:     2,
 				AssessmentKeyFilesProblems:     3,
-				AssessmentKeyResponseNoExcess:  4,
+				AssessmentKeyResponseNoError:   4,
+				AssessmentKeyResponseNoExcess:  5,
+				AssessmentKeyResponseNotEmpty:  6,
+				AssessmentKeyResponseWithCode:  7,
 			},
 			"ModelB": Assessments{
 				AssessmentKeyCoverageStatement: 1,
 				AssessmentKeyFilesExecuted:     2,
 				AssessmentKeyFilesProblems:     3,
-				AssessmentKeyResponseNoExcess:  4,
+				AssessmentKeyResponseNoError:   4,
+				AssessmentKeyResponseNoExcess:  5,
+				AssessmentKeyResponseNotEmpty:  6,
+				AssessmentKeyResponseWithCode:  7,
 			},
 		},
 
 		ExpectedString: `
-			model,files-executed,files-problems,coverage-statement,response-no-excess
-			ModelA,2,3,1,4
-			ModelB,2,3,1,4
+			model,files-executed,files-problems,coverage-statement,response-no-error,response-not-empty,response-with-code,response-no-excess
+			ModelA,2,3,1,4,6,7,5
+			ModelB,2,3,1,4,6,7,5
 		`,
 	})
 }
+
+func TestAssessmentsIsEqual(t *testing.T) {
```

Comment:
https://github.com/symflower/eval-dev-quality/pull/39/commits/a1d5ee333f333310d1af796e6a6c727c12bf8954#r1570186697

Reply:
linking to comments does not work at all for me... it always sends me to the whole changes... I think the link might only work once the review is sent?

Reply:
https://github.com/symflower/eval-dev-quality/pull/39#discussion_r1570186697

Reply:
Looks like it. but i always send the comment right away. i do not use the start-review-functionality. because i hate that even more.